plan/logical_plan_test.go

```diff
@@ -660,6 +660,28 @@ func (s *testPlanSuite) TestPlanBuilder(c *C) {
 	}
 }
 
+func (s *testPlanSuite) TestPlanBuilderError(c *C) {
```

Comment:
We have a test called `TestValidate`. I'm sorry I made a mistake in last comment.

Reply:
Move ErrKeyDoesNotExits check ut to `TestValidate` in 98d084b

Sorry, I am not very familiar with the case organization.